js/router.js

```diff
@@ -48,6 +48,10 @@ module.exports = Backbone.Router.extend({
         this.navigate(translatedRoute, { trigger: true });
       });
     });
+    
+    history.size = -1;
```

Comment:
Why are you adding these parameters to the history object instead of an object like this.routeHistory?